packages/wonder-blocks-testing/package.json

```diff
@@ -20,6 +20,7 @@
         "@khanacademy/wonder-stuff-core": "^0.1.2",
         "@khanacademy/wonder-stuff-testing": "^0.0.2",
         "@storybook/addon-actions": "^6.4.8",
+        "node-fetch": "^2.6.7",
```

Comment:
Peer dependency for when using in Jest environment that is not JSDOM (and we're not using a version of Node with the fetch API - coming with v17+).